bucket/mill.json

```
@@ -1,9 +1,10 @@
 {
+    "homepage": "http://www.lihaoyi.com/mill/",
+    "description": "Your shiny new Java/Scala build tool!",
```

Nice example of how description should not looks like 😀

```suggestion
    "description": "Build tool aiming for Java and Scala.",
```

---

Agreed. Again, I was just cut-and-pasting from their website.